docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rust:1.80.0-alpine AS builder
+RUN apk upgrade --no-cache
+RUN apk add --update --no-cache \
+    pkgconfig \
+    musl-dev \
+    openssl-dev
+WORKDIR /build
+COPY ./src ./src
+COPY ./Cargo.toml ./Cargo.toml
+RUN RUSTFLAGS='-C target-feature=-crt-static' cargo build --release
```

Comment:
Please check it, possible no more needed

Reply:
This is necessary to build rust library on alpine